code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -137,40 +103,23 @@ Please contact me on #coderbus IRC. ~Carn x
 	if(QDESTROYING(src))
 		return
 
-	lying_prev = lying	//so we don't update overlays for lying/standing unless our stance changes again
-	update_hud()		//TODO: remove the need for this
+	crash_with("CANARY: Old human update_icons was called.")
 
-	//0: We start with their existing appearance (this contains their verbs, important to keep those!)
-	var/mutable_appearance/ma_compiled = new(src)
-
-	//1: HUDs because these are hidden behind a backplane. See update_icons_huds()
-	ma_compiled.overlays = list_huds //The first one can set instead of add
-
-	//2: The body itself, all the organs and whatnot
-	ma_compiled.overlays += list_body
-
-	//3: The 'layers' list (overlays_standing), from the defines above
-	ma_compiled.overlays += list_layers
-
-	//4: Apply transforms based on situation
-	update_transform(ma_compiled)
+	update_hud()		//TODO: remove the need for this
 
-	//5: Do any species specific layering updates, such as when hiding.
-	update_icon_special(ma_compiled, FALSE)
+	//Do any species specific layering updates, such as when hiding.
+	update_icon_special()
 
-	//6: Set appearance once
-	appearance = ma_compiled
+/mob/living/carbon/human/update_icons_layers()
+	crash_with("CANARY: Old human update_icons was called.")

```

Comment:
Looks like you meant to put update_icons_layers here

Reply:
Can fix, but I'd rather wait for more review comments so I can do it in a big fix commit.

Reply:
That's exactly the thing I expected you to say, dun worry ;>